### PR TITLE
494/tweaks bugs

### PR DESCRIPTION
**Pull request by sintinilorenzo**

### CODE REVIEW

File: src/components/organisms/AppRouter/AppRouter.jsx [hunk 1]
```diff
@@ -81,7 +81,15 @@ const AppRouter = ({ defaultVenue }) => {
         />
         <Route
           path="/"
-          component={() => <Redirect to={venueLandingUrl(defaultVenue)} />}
+          component={() => (
+            <Redirect
+              to={
+                defaultVenue === "playa"
```
Comment (thecadams) [inline src/components/organisms/AppRouter/AppRouter.jsx:87]: let’s supply a `defaultRedirect` to AppRouter and move this logic there, so there’s only one place in the code with this special logic. So index.tsx would be similar to the existing code but bring the venueLandingUrl/venueInsideUrl into it, something like..

const defaultRedirect = projectId === “co-reality-map” ? venueLandingUrl(“kansassmittys”) : venueInsideUrl(“playa”)

File: src/pages/Admin/Venue/DetailsForm.tsx [hunk 2]
```diff
@@ -80,7 +80,8 @@ export const DetailsForm: React.FC<DetailsFormProps> = ({
         // unfortunately the typing is off for react-hook-forms.
         if (editing) await updateVenue(vals as VenueInput, user);
         else await createVenue(vals as VenueInput, user);
-        history.push("/admin");
+        //history.push("/admin");
```
Comment (thecadams) [inline src/pages/Admin/Venue/DetailsForm.tsx:83]: nit: commented code

File: src/components/organisms/AppRouter/AppRouter.jsx [hunk 3]
```diff
@@ -81,7 +81,15 @@ const AppRouter = ({ defaultVenue }) => {
...
+                  ? venueInsideUrl(defaultVenue)
```
Comment (mathieud) [inline src/components/organisms/AppRouter/AppRouter.jsx:88]: I think Chris did that already: https://github.com/sparkletown/sparkle/pull/329

File: src/pages/Admin/Venue/DetailsForm.tsx (hunk 2 above)
Comment (mathieud) [inline src/pages/Admin/Venue/DetailsForm.tsx:83]: Remove the commented line

File: src/components/organisms/AppRouter/AppRouter.jsx (hunk 3 above)
Comment (sintinilorenzo) [reply src/components/organisms/AppRouter/AppRouter.jsx:88]: it was venueLandingUrl(defaultVenue) which sends you to "v/playa". For the playa we need to go to "in/playa" with the venueInsideUrl(defaultVenue).

File: src/index.tsx [hunk 4]
```diff
@@ -105,18 +104,15 @@ const AuthIsLoaded: React.FunctionComponent<React.PropsWithChildren<{}>> = ({
...
+const projectID = firebaseConfig.projectId;
 
 render(
   <Elements stripe={stripePromise}>
     <DndProvider backend={HTML5Backend}>
       <Provider store={store}>
         <ReactReduxFirebaseProvider {...rrfProps}>
           <AuthIsLoaded>
-            <AppRouter defaultRedirect={defaultRedirect} />
+            <AppRouter projectID={projectID} />
```
Comment (thecadams) [inline src/index.tsx:115]: This should still be defaultRedirect

**Pull request merged**